gruntfileConfig.json

```diff
@@ -4,6 +4,7 @@
   "src": "./src",
   "assets": "./src/assets",
   "theme": "./node_modules/patternpack-example-theme",
+  "logo": "/theme-assets/images/pp-logo.svg",
```

Comment:
Recapping a phone conversation:

1. Rename logo to logo.svg
2. We will keep as .svg with the assumption that if a developer wants to provide a default logo + the ability to customize it, the logo will have to be `/theme-assets/images/logo.svg`